### PR TITLE
SoundWire: cleanup baseline

### PR DESCRIPTION
**Pull request by plbossart**
Before we start, let's clean this up.

@xiulipan can you please add soundwire-defconfig to the SOF and SST options for the kernel CI? Thanks!

### CODE REVIEW

File: drivers/soundwire/cadence_master.c [hunk 1]
```diff
@@ -306,8 +306,8 @@ _cdns_xfer_msg(struct sdw_cdns *cdns, struct sdw_msg *msg, int cmd,
 	return cdns_fill_msg_resp(cdns, msg, count, offset);
 }
 
-static enum sdw_command_response cdns_program_scp_addr(
-			struct sdw_cdns *cdns, struct sdw_msg *msg)
+static enum sdw_command_response
+cdns_program_scp_addr(struct sdw_cdns *cdns, struct sdw_msg *msg)
```
Comment (ranj063) [inline drivers/soundwire/cadence_master.c:310]: @plbossart this looks odd. should the function name be on the previous line?

Comment (plbossart) [reply drivers/soundwire/cadence_master.c:310]: it's common practice, even in this file, to put the type first when the entire function doesn't fit on one line.

Comment (ranj063) [reply drivers/soundwire/cadence_master.c:310]: ok

**Comment by plbossart** [comment]
will submit upstream first, additional changes needed

**Comment by xiulipan** [comment]
@plbossart 
Will check that and try to add in sof-ci first.

**Pull request closed without merging**